chain/ethereum/src/chain.rs

```diff
@@ -169,57 +292,33 @@ impl Blockchain for Chain {
         metrics: Arc<BlockStreamMetrics>,
         unified_api_version: UnifiedMappingApiVersion,
     ) -> Result<Box<dyn BlockStream<Self>>, Error> {
-        let logger = self
-            .logger_factory
-            .subgraph_logger(&deployment)
-            .new(o!("component" => "BlockStream"));
-        let chain_store = self.chain_store().clone();
-        let writable = self
-            .subgraph_store
-            .writable(&deployment)
-            .expect(&format!("no store for deployment `{}`", deployment.hash));
-        let chain_head_update_stream = self.chain_head_update_listener.subscribe(self.name.clone());
-
-        let requirements = filter.node_capabilities();
-
-        let triggers_adapter = self
-            .triggers_adapter(
-                &deployment,
-                &requirements,
-                unified_api_version.clone(),
-                metrics.stopwatch.clone(),
-            )
-            .expect(&format!(
-                "no adapter for network {} with capabilities {}",
-                self.name, requirements
-            ));
-
-        // Special case: Detect Celo and set the threshold to 0, so that eth_getLogs is always used.
-        // This is ok because Celo blocks are always final. And we _need_ to do this because
-        // some events appear only in eth_getLogs but not in transaction receipts.
-        // See also ca0edc58-0ec5-4c89-a7dd-2241797f5e50.
-        let chain_id = self.eth_adapters.cheapest().unwrap().chain_id().await?;
-        let reorg_threshold = match CELO_CHAIN_IDS.contains(&chain_id) {
-            false => self.reorg_threshold,
-            true => 0,
-        };
+        let favor_firehose = env::var("FIREHOSE_PRIORITY");
```

Comment:
It would be better to map to a boolean right here to make the next line a little clearer.